components/director/internal/open_resource_discovery/ord_document.go

```diff
@@ -589,3 +619,14 @@ func stringPtrToString(p *string) string {
 	}
 	return ""
 }
+
+func deleteInvalidInputObjects[T any](invalidObjectsIndices []int, objects []T) []T {
+	decreaseIndexForDeleting := 0
+	for _, eventIndex := range invalidObjectsIndices {
```

Comment:
maybe consider renaming `eventIndex` to `invalidObjectIdx` ?

Reply:
That was a leftover from previous implementation, thanks for the observation I changed the name.